docs/sources/configuration/perception.rst

```diff
@@ -0,0 +1,51 @@
+Perception
+============
+
+The collision avoidance depends mostly on the Configuration > Perception dialog where the connection and description of either a laserscan sensor or an obstacle map message is configured.
+
```

Comment:
Can we improve this statement. "depends mostly on the Configuration > Perception dialog".  Maybe: "Collision avoidance uses settings in this dialog to describe the sensors used during operation."